x-pack/plugins/canvas/public/lib/aeroelastic/store.ts

```diff
@@ -8,7 +8,13 @@ import { ActionId, Payload, State, TypeName, UpdaterFunction } from '.';
 
 let counter = 0 as ActionId;
 
-export const createStore = (initialState: State, updater: UpdaterFunction) => {
+interface Store {
+  getCurrentState: () => State;
```

Comment:
couldn't resist a quick look 😄 
wondering if we should put this in `index.d.ts`

Reply:
Sure, no problem.